recycle-node.rb

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env ruby
+
+
+# Usage:
+#   export AWS_PROFILE=moj-cp
+#   Edit "K8S_CLUSTER_NAME" to specify the cluster to recycle old node. 
+#   Set kubectl config to use-context #K8S_CLUSTER_NAME
+#   ./create-cluster.rb
+#
+#  Note: To run this on a test cluster, update function `get_worker_instance_group_size` as shown below, 
```

Comment:
Please add some explanation of why this is necessary.

Something like, "The method looks for the instance group size in the kops manifest in our github repository, so it will always return the size of the live-1 cluster..."